### DISCUSSION TITLE
Blue smart ip22 und 100/30 mppt gleichzeitig laden

### DISCUSSION THREAD

**Discussion by Michel83** [hoylabs/OpenDTU-OnBattery#1048] [General]
Hi,

hat diese Konstellation vielleicht jemand wie oben beschrieben oder kann mir sagen ob diese mit open dtu on battery funktioniert.
Im Victron forum habe ich gefunden das es funktioniert und die beiden Geräte einfach über Bluetooth gekoppelt werden, geht das weil mein mppt ja schon an der open dtu mit dem ve. Netzwerk hängt.
Grüsse

**Comment by SW-Niko** [comment]
Hallo @Michel83,
ich verwende keinen "Blue Smart IP22". Aber ich habe 2 MPPT und eine "Smart Batterie Sense" über Bluetooth gekoppelt am laufen.
Bis jetzt holt sich OpenDTU-OnBattery nur Daten und Informationen von dem Victron-Geräten. (Keine Steuerung)
Darum denke ich das es problemlos funktionieren sollte.

**Reply by Michel83** [reply]
Hi, danke für die Antwort dann werde ich wohl mal einen Blue Smart kaufen.

Danke nochmal.

Grüsse
